terraform/modules/aws-cargohold-asset-storage/README.md

```diff
@@ -0,0 +1,17 @@
+Terraform module: Cargohold Asset Storage
+=========================================
+
+This module creates an Object Storage on AWS for cargohold to store encrypted assets.
```

Comment:
This module is experimental, and lacks cloudfront setup. Can you add this to the readme? (see also the other readmes)